### PR TITLE
Task6

### PR DESCRIPTION
**Pull request by igorano**

### CODE REVIEW

File: src/com/estafet/learning/US006/Task6.java [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+    static float[] floatArr;
+
+    private static void intArray() {
+        floatArr = new float[10];
+        int[] intArr = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
+
+        System.out.println("\n------------------------");
+        System.out.println("Array length is: " + intArr.length);
+        for (int i = 0; i <= 9; i++) {
```
Comment (azankovskiy) [inline src/com/estafet/learning/US006/Task6.java]: hardcoded array length

File: src/com/estafet/learning/US006/Task6.java [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+        System.out.println("\nArray with " + x + " added:\n" + Arrays.toString(arr));
+        System.out.println("\n------------------------");
+    }
+
+    private static void findDuplicateValuesOfArray() {
+        int[] arr = new int[]{5, 1, 2, 3, 4, 2, 7, 8, 8, 3, 5};
+        System.out.println("\n------------------------");
+        System.out.println("\nDuplicate elements in the given array: ");
+        for (int i = 0; i < arr.length; i++) {
+            for (int j = i + 1; j < arr.length; j++) {
```
Comment (azankovskiy) [inline src/com/estafet/learning/US006/Task6.java]: Optimization problem: what if array will contain a million of elements - will that work?

File: src/com/estafet/learning/US006/Task6.java (hunk 1 above)
Comment (igorano) [reply src/com/estafet/learning/US006/Task6.java]: changed

File: src/com/estafet/learning/US006/Task6.java (hunk 2 above)
Comment (igorano) [reply src/com/estafet/learning/US006/Task6.java]: changed.

**Pull request merged**